platformio.ini

```diff
@@ -14,7 +14,7 @@ default_envs = debug
 [env]
 ; If Platformio gives a upload error, please specify this for your respective COM port
 upload_speed = 230400
-
+upload_port = /dev/ttyUSB0
```

Comment:
Isn't the upload port automatically detected?

Reply:
There were some issues with this on exo.
.

Reply:
The port detected on exo is the port of the on board arduino and not this one, so we have to specify this